meta/meta-node/Cargo.toml

```diff
@@ -54,8 +59,20 @@ fp-evm							= { git = "https://github.com/paritytech/frontier.git", branch = "p
 fp-rpc							= { git = "https://github.com/paritytech/frontier.git", branch = "polkadot-v0.9.27" }
 fp-storage						= { git = "https://github.com/paritytech/frontier.git", branch = "polkadot-v0.9.27" }
 # local packages
-meta-runtime = { package = "meta-runtime", path = "../meta-runtime" }
+meta-runtime = { default-features = false, features = ["std"], package = "meta-runtime", path = "../meta-runtime" }
```

Comment:
I think this might cause issues with code coverage. Since the code coverage features are disabled?